library/src/scripts/components/headers/VanillaHeader.tsx

```diff
@@ -94,6 +95,16 @@ export class VanillaHeader extends React.Component<IProps, IState> {
         const containerElement = this.props.container || document.getElementById("vanillaHeader")!;
         containerElement.classList.value = outerCssClasses;
 
+        const themeLogo = getLogo(LogoType.DESKTOP);
```

Comment:
This breaking the encapsulation we get from redux, which we shouldn't do unless we have to. Right now the logo wouldn't update for example if it changed in the store.

It should probably come in through some `mapStateToProps`.